Sources/SwiftParser/Lexer/Cursor.swift

```diff
@@ -433,28 +445,24 @@ extension Lexer.Cursor {
     if let stateTransition = result.stateTransition {
       self.stateStack.perform(stateTransition: stateTransition, stateAllocator: stateAllocator)
     }
-
+    
```

Comment:
I think `swift-format` will complain about this whitespace. Could you run `format.py`? https://github.com/apple/swift-syntax/blob/main/CONTRIBUTING.md#formatting

Reply:
Oops, I missed that!
I'll make sure to run it before every commit.
Thanks for pointing that out 🙇